test/integration/targets/ios_acl/tests/cli/rtt.yaml

```diff
@@ -0,0 +1,173 @@
+                    destination:
+                      address: 192.0.4.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: www
+                    dscp: ef
+                    ttl:
+                      lt: 20
+          - afi: ipv6
+            acls:
+              - name: R1_TRAFFIC
+                aces:
+                  - grant: deny
+                    protocol_options:
+                      tcp:
+                        ack: true
+                    source:
+                      any: true
+                      port_protocol:
+                        eq: www
+                    destination:
+                      any: true
+                      port_protocol:
+                        eq: telnet
+                    dscp: af11
+        state: merged
+      register: base_config
+
+    - name: Gather static routes facts
+      ios_facts:
+        gather_subset:
+          - "!all"
+          - "!min"
+        gather_network_resources:
+          - acl
+
+    - name: Apply the configuration which need to be reverted
+      ios_acl:
+        config:
+          - afi: ipv4
+            acls:
+              - name: 110
+                aces:
+                  - grant: deny
+                    protocol_options:
+                      tcp:
+                        ack: true
+                    source:
+                      address: 198.51.100.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    destination:
+                      address: 198.51.110.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: www
+                    dscp: ef
+                    ttl:
+                      eq: 10
+              - name: 150
+                aces:
+                  - grant: deny
+                    protocol_options:
+                      tcp:
+                        syn: true
+                    source:
+                      address: 198.51.100.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    destination:
+                      address: 198.51.110.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    dscp: ef
+                    ttl:
+                      eq: 10
+        state: overridden
+      register: result
+
+    - assert:
+        that:
+          - "result.commands|length == 8"
+          - "result.changed == true"
+          - "result.commands|symmetric_difference(overridden.commands) == []"
+
+    - name: Revert back to base config using facts round trip
+      ios_acl:
+        config: "{{ ansible_facts['network_resources']['acl'] }}"
+        state: overridden
+      register: revert
+
+    - assert:
```

Comment:
Please assert the before and after list of dicts.

Reply:
same as above